### PR TITLE
add custom domain name

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn what to do about the cms? should it also have a field for custom dns or should it just be `cms.{custom_dns}`?

**Comment by smn** [comment]
Hm. For the CMS we can do the same if they want to I suppose? But let's land this first.
My only remark on this is that the name of the variable is potentially going to be confusing, we're providing a custom domain name, not a custom domain name server.

**Comment by miltontony** [comment]
Let me rename quickly

**Comment by SteveBarnett** [comment]
@miltontony @smn My 2c: it would great to have custom DNS for the CMS too, so that users can set it to something they're familiar with. That might be `cms`, `admin` - something they're already familiar with from other CMSes.

**Comment by miltontony** [comment]
Cool. In another PR I'll add `frontend_custom_domain` and `cms_custom_domain`

**Comment by miltontony** [comment]
@smn re-review please. I've renamed the field

File: unicoremc/views.py [hunk 1]
```diff
@@ -75,6 +75,7 @@ def form_valid(self, form):
 
         project = self.get_object()
         project.create_pyramid_settings()
+        project.create_nginx()
```
Comment (smn) [inline unicoremc/views.py:78]: eh? Where did this happen before?

**Comment by smn** [comment]
:+1: but just noting that coveralls is a bit unhappy

File: unicoremc/views.py (hunk 1 above)
Comment (miltontony) [inline unicoremc/views.py:78]: This usally happens in the workflow when a new project is created. We never needed to recreate nginx configs in the views before, only the pyramid settings.

**Pull request merged**